oe_theme.theme

```diff
@@ -1858,6 +1861,39 @@ function _oe_theme_preprocess_site_specific_footer_links(array &$variables): voi
   }
 }
 
+/**
+ * Get the Accessibility statement link.
+ */
+function _oe_theme_preprocess_accessibility_link(array &$variables) {
+  $cacheability = CacheableMetadata::createFromRenderArray($variables);
+  $configuration = \Drupal::configFactory()->get(SiteInformation::CONFIG_NAME);
+  $cacheability->addCacheableDependency($configuration);
+  /** @var \Drupal\oe_corporate_site_info\SiteInformationInterface $site_information */
+  $site_information = \Drupal::service('oe_corporate_site_info.site_information');
```

Comment:
We need to check if the module is enabled before we get the service, the module is added as require_dev so not all sites will have it.